vertx-db2-client/pom.xml

```diff
@@ -34,6 +34,7 @@
   <properties>
     <doc.skip>false</doc.skip>
     <docs.dir>${project.basedir}/src/main/docs</docs.dir>
+    <jar.manifest>${project.basedir}/src/main/resources/META-INF/MANIFEST.MF</jar.manifest>
```

Comment:
can we try having this in the parent pom of the project using inheritance ?

Reply:
done